cnames_active.js

```diff
@@ -1753,6 +1753,7 @@ var cnames_active = {
   "router-advanced": "oldergod.github.io/router-advanced",
   "router5": "hosting.gitbook.com",
   "routex": "routex.netlify.com",
+  "roxza": "roxza.glitch.me", // Pls Glitch
```

Comment:
```suggestion
  "roxza": "roxza.glitch.me",
```
Please remove the coomment from your entry.